src/clients/python/ci.zig

```diff
@@ -0,0 +1,116 @@
+const std = @import("std");
+const builtin = @import("builtin");
+const log = std.log;
+const assert = std.debug.assert;
+
+const flags = @import("../../flags.zig");
+const fatal = flags.fatal;
+const Shell = @import("../../shell.zig");
+const TmpTigerBeetle = @import("../../testing/tmp_tigerbeetle.zig");
+
+pub fn tests(shell: *Shell, gpa: std.mem.Allocator) !void {
+    assert(shell.file_exists("pyproject.toml"));
+
+    // Integration tests.
+
+    // `python -m build` won't compile the native library automatically, we need to do that
+    // ourselves.
+    try shell.exec_zig("build clients:python -Drelease", .{});
+
+    // We need to build the tigerbeetle_python library manually for samples/testers to work.
+    shell.cwd.makeDir("src/tigerbeetle_python/lib") catch {};
+    var lib_dir = try shell.cwd.openDir("src/tigerbeetle_python/lib", .{});
+    defer lib_dir.close();
+    {
+        try shell.pushd("../c/lib");
+        defer shell.popd();
+
+        var copied_count: u32 = 0;
+        const native_files = try shell.find(.{
+            .where = &.{"."},
+            .extensions = &.{ ".dll", ".so", ".dylib" },
+        });
+        copied_count = 0;
+        for (native_files) |native_file| {
+            try Shell.copy_path(shell.cwd, native_file, lib_dir, native_file);
+            copied_count += 1;
+        }
+        // 7 = 4 + 3
+        //     4 = x86_64 for mac, windows and {linux musl, linux glibc}
+        //         3 = aarch64 for mac and {linux musl, linux glibc}
+        assert(copied_count == 7);
```

Comment:
Yeah, I've duplicated building the shared library into `build_python_client` rather than depending on the `c_client` step... But I'm not sure on it: the existing clients build their own libraries whereas the Python one does really use the C client libraries directly.

The C sample builds the C library directly, so I took inspiration from that.